apps/design-land/src/app/card/card.module.ts

```diff
@@ -29,29 +13,8 @@ import { ReactiveFormsModule } from '@angular/forms';
     CommonModule,
     DesignLandCardRoutingModule,
     DesignLandExampleViewerModule,
-    ReactiveFormsModule,
-    DaffCardModule,
-    DaffImageModule,
-    CardExamplesModule,
   ],
 })
 export class CardModule {
```

Comment:
Can we update this to `DesignLandCardModule` and do it in the component too since you started doing that to other components in the PR?

Reply:
fixed.

Reply:
I've backed these changes out and will submit in another PR.